code/modules/mob/living/login.dm

```diff
@@ -6,8 +6,9 @@
 	update_sight()
 
 	//If they're SSD, remove it so they can wake back up.
+	if(player_logged && istype(src, /mob/living/carbon/human))
```

Comment:
This is disgusting and an *egregious* violation of OOP. Don't do this.

Reply:
so what should I do instead?

Reply:
put it under the object that you want to have the behavior.